src-java/flowhs-topology/flowhs-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowUpdatePayload.java

```diff
@@ -71,10 +73,11 @@ public FlowUpdatePayload(@JsonProperty(Utils.FLOW_ID) String id,
                              @JsonProperty("priority") Integer priority,
                              @JsonProperty("pinned") Boolean pinned,
                              @JsonProperty("encapsulation-type") String encapsulationType,
-                             @JsonProperty("path-computation-strategy") String pathComputationStrategy) {
+                             @JsonProperty("path-computation-strategy") String pathComputationStrategy,
+                             @JsonProperty("statistics-vlan") Set<Integer> vlanStatistics) {
         super(id, source, destination, maximumBandwidth, ignoreBandwidth, periodicPings, allocateProtectedPath,
                 description, created, lastUpdated, status, maxLatency, priority, pinned, encapsulationType,
-                pathComputationStrategy);
+                pathComputationStrategy, vlanStatistics);
```

Comment:
This class is used in v1 API, But we need to change only v2 API. please remove it

Reply:
Fixed